conf/dhcpcd-hooks/70-kano-updater-download

```diff
@@ -0,0 +1,32 @@
+# 70-kano-updater-download
```

Comment:
Was wondering if you needed a `#!` - it turns out `dhcpcd-run-hooks`  _sources_ these scripts; yuck.